### PR TITLE
refactor(test): refactor log, dashboard, me

### PR DESCRIPTION
**Pull request by simeng-li**
refactor log, dashboard, me

## Summary

Refactor log, dashboard, and me test cases.

## Testing

test locally
@logto-io/eng @xiaoyijun

### CODE REVIEW

**Review by wangsijie — APPROVED** [review]

File: packages/integration-tests/tests/me.test.ts [hunk 1]
```diff
@@ -20,56 +25,30 @@ describe('api `/me`', () => {
...
 
-    const headers = Object.freeze({
-      'development-user-id': user.id,
-    });
+    const foo = 'bar';
 
-    const newCustomData = {
-      foo: 'bar',
-    };
+    await updateCurrentUserCustomDate(user.id, { foo });
```
Comment (xiaoyijun) [inline packages/integration-tests/tests/me.test.ts:37]:
```suggestion
    await updateCurrentUserCustomData(user.id, { foo });
```

File: packages/integration-tests/src/api/me.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+export const getCurrentUserCustomData = (userId: string) =>
+  api
+    .get('me/custom-data', {
+      headers: {
+        'development-user-id': userId,
+      },
+    })
+    .json<ArbitraryObject>();
+
+export const updateCurrentUserCustomDate = (userId: string, payload: Record<string, unknown>) =>
```
Comment (xiaoyijun) [inline packages/integration-tests/src/api/me.ts:17]:
```suggestion
export const updateCurrentUserCustomData = (userId: string, payload: Record<string, unknown>) =>
```

**Review by xiaoyijun — APPROVED** [review]
Overall LGTM.

File: packages/integration-tests/tests/me.test.ts [hunk 3]
```diff
@@ -1,15 +1,20 @@
...
-
-import { api } from '@/api';
-import { createUserByAdmin } from '@/helpers';
+import { userInfoSelectFields } from '@logto/schemas';
+import { assert } from '@silverhand/essentials';
+
+import {
+  getCurrentUserInfo,
+  getCurrentUserCustomData,
+  updateCurrentUserCustomDate,
```
Comment (xiaoyijun) [inline packages/integration-tests/tests/me.test.ts:7]:
```suggestion
  updateCurrentUserCustomData,
```

File: packages/integration-tests/tests/dashboard.test.ts [hunk 4]
```diff
@@ -50,31 +32,20 @@ describe('admin console dashboard', () => {
...
-      mau: StatisticsData;
-    };
-
-    const originActiveUserStatistics = await authedAdminApi
-      .get('dashboard/users/active')
-      .json<ActiveUserStatistics>();
+    const originActiveUserStatistics = await getActiveUsersData();
 
-    await registerUserAndSignIn();
+    await signIn(username, password);
```
Comment (xiaoyijun) [inline packages/integration-tests/tests/dashboard.test.ts:37]: This test relies on the test above (register user); per discussion on [slack](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1658315031191559), should we keep them independent?

Comment (simeng-li) [reply packages/integration-tests/tests/dashboard.test.ts:40]: should be good after making the whole test process linear. Anyway updated.

**Pull request merged**